src/chapters/07/main.tex

```diff
@@ -227,21 +215,25 @@ \section{Solving with Python}\label{sec:solving-with-python}
 The \mintinline{python}{__init__} method is run whenever the object is first
 created, and initialises the object.
 In this case it sets a number of attributes.
-First the square grid's \mintinline{python}{size} is defined, which is the
-number of rows and columns of houses it contains.
-Next we define \mintinline{python}{coords}, a list of tuples representing all
-the possible coordinates of the grid, this uses the
-\mintinline{python}{itertools} library for efficient looping.
-Finally \mintinline{python}{houses} is defined, a dictionary with grid
-coordinates as keys, and instances of the, yet to be defined,
-\mintinline{python}{House} class representing the houses themselves.
+
+\begin{itemize}
+     \item First the square grid's \mintinline{python}{size} is defined, which
+           is the number of rows and columns of houses it contains.
+     \item Next the \mintinline{python}{coords} are defined, a list of tuples
+           representing all the possible coordinates of the grid, this uses the
+           \mintinline{python}{itertools} library for efficient iteration.
+     \item Finally \mintinline{python}{houses} is defined, a dictionary with
+           grid coordinates as keys, and instances of the
+           \mintinline{python}{House} class.
+\end{itemize}
+
 
 The \mintinline{python}{run} method runs the simulation. For each
 \mintinline{python}{n_steps} number of discrete time steps, the city runs the
 method \mintinline{python}{take_turn}.
 In this method, we first create a list of all the houses with households that
 are unhappy with their neighbours; these are put in a random order using the
-\mintinline{python}{random} library; and then working inwards from the ends,
+\mintinline{python}{random} library; and then working inwards from the boundary
```

Comment:
I don't think `boundary` is the right word. It the ends of a list, not the boundary of the city. What do you think @drvinceknight ?

Reply:
How about: `from the inner and outer most elements` ?

Reply:
yeah happy with that